composer.json

```diff
@@ -13,16 +13,26 @@
         }
     },
     "require": {
-        "symfony/symfony": "3.2.*"
+        "symfony/framework-bundle": "^3.2|^4.0",
```

Comment:
@jmwamser How can we test this library with `Symfony 3`? :thinking: I've tried to run `composer update` on your branch, but I had some errors because of the PHP version. Thanks.